### PR TITLE
add -v flag to spin and kick off integration testing

### PR DESCRIPTION
**Pull request by grosser**
this is mainly a first step to integration testing then a new feature :)
I'll try to continue on this path and spec out the most important parts of the workflow

### CODE REVIEW

File: bin/spin [hunk 1]
```diff
@@ -15,6 +15,8 @@ require 'benchmark'
 require 'optparse'
 require 'pathname'
 
+$LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
```
Comment (jstorimer) [inline bin/spin]: This shouldn't be necessary when it's installed as a gem. Should be removed.

File: spin.gemspec [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+$LOAD_PATH.unshift File.expand_path("../lib", __FILE__)
+require "spin/version"
+
+Gem::Specification.new "spin", Spin::VERSION do |s|
   s.authors     = ["Jesse Storimer"]
   s.email       = ["jstorimer@gmail.com"]
   s.homepage    = "http://jstorimer.github.com/spin"
   s.summary     = %q{Spin preloads your Rails environment to speed up your autotest(ish) workflow.}
-  s.description = %q{Spin preloads your Rails environment to speed up your autotest(ish) workflow.
+  s.description = %Q{#{s.summary}
```
Comment (jstorimer) [inline spin.gemspec:9]: I think you meant:

``` ruby
s.description = s.summary
```

File: bin/spin (hunk 1 above)
Comment (jstorimer) [inline bin/spin]: If this is needed for the test then lets add it there instead.

File: spin.gemspec (hunk 2 above)
Comment (grosser) [inline spin.gemspec:9]: old description was summary + more, just replaced the summary part

File: bin/spin (hunk 1 above)
Comment (grosser) [inline bin/spin]: Yeah, found a workaround, already used this pattern in ~5 gems and can finally retire it :)

**Comment by grosser** [comment]
@jstorimer are you ok with me merging this ?

**Comment by jstorimer** [comment]
:shipit:

**Pull request merged**